pulsar-common/src/main/java/org/apache/pulsar/common/util/FutureUtil.java

```diff
@@ -258,4 +259,25 @@ public static CompletionException wrapToCompletionException(Throwable throwable)
             return new CompletionException(throwable);
         }
     }
+
+    /**
+     * Executes an operation using the supplied {@link Executor}
+     * and notify failures on the supplied {@link CompletableFuture}.
+     *
+     * @param runnable the runnable to execute
+     * @param executor  the executor to use for executing the runnable
+     * @param completableFuture  the future to complete in case of exceptions
+     * @return
+     */
+
+    public static void safeRunAsync(Runnable runnable,
```

Comment:
It could improve readability of the code using this if the Runnable is the last argument. However, I don't have a strong opinion about this.

Reply:
I like this order because it's the same of `CompletableFuture#runAsync(Runnable runnable, Executor executor)`